benchmarks/Dapper.Tests.Performance/Dapper.Tests.Performance.csproj

```diff
@@ -46,9 +47,20 @@
     <Reference Include="Microsoft.CSharp" />
     <Reference Include="System.Configuration" />
     <Reference Include="System.Data.Linq" />
+    <Compile Remove="Benchmarks.Nextorm.cs" />
   </ItemGroup>
   <ItemGroup Condition="'$(TargetFramework)' != 'net462'">
-    <PackageReference Include="Microsoft.EntityFrameworkCore.SqlServer" />
     <PackageReference Include="Norm.net" />
   </ItemGroup>
+  <ItemGroup Condition="'$(TargetFramework)' == 'net5.0'">
+    <PackageReference Include="Microsoft.EntityFrameworkCore.SqlServer" VersionOverride="5.0.17" />
+    <Compile Remove="Benchmarks.Nextorm.cs" />
+  </ItemGroup>
+  <ItemGroup Condition="'$(TargetFramework)' == 'net8.0'">
+    <PackageReference Include="nextorm.sqlserver" />
+    <PackageReference Include="Microsoft.EntityFrameworkCore.SqlServer" VersionOverride="8.0.0" />
```

Comment:
why do we need all these version-overrides? that seems unlikely

Reply:
I've got this
`error NU1010: The PackageReference items Microsoft.Extensions.Logging.Console;Microsoft.Extensions.ObjectPool do n 
ot have corresponding PackageVersion`

Reply:
that probably means it needs adding to `Directory.Packages.props`

Reply:
Done